README.md

```diff
@@ -1,253 +1,46 @@
-
-<!-- README.md is generated from README.Rmd. Please edit that file -->
```

Comment:
Rendering a `README.Rmd` adds a lot of extra complexity and leads to derivative files in the git repo.
Instead, I've now moved the general pkg introduction to the `openairegraph.Rmd` vignette, which pkgdown will highlight on the website by default.
Vignettes always get rendered and published on every build, so that's one less thing to worry about and the repo stays clean.